modules/RsPaymentTerminal/src/main.rs

```diff
@@ -154,6 +164,13 @@ pub struct PaymentTerminalModule {
 
     /// The Feig interface.
     feig: SyncFeig,
+
+    /// For which connectors credit cards should be accepted
+    /// If None, authentication happens to all connectors
+    credit_cards_connectors: Vec<i64>,
```

Comment:
None can be only true for Option. Maybe lets make it an option?

Reply:
It doesn't have to. It's a config - it should always be set to something. Even empty

Reply:
I mean in the comment above